### PR TITLE
create migrations for user and shop tables

### PR DESCRIPTION
**Pull request by asadbek21iman**
create migrations for user and shop tables

### CODE REVIEW

**Comment by asadbek21iman** [comment]
i have updated PR so that it would contain product migrations as well

File: migrations/000001_users.down.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS "users";
```
Comment (khudayberdiyev25) [inline migrations/000001_users.down.sql:1]: add extra line at the end of the file to get rid of this alert

File: migrations/000002_shop.up.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE "shop" (
+  "id" SERIAL PRIMARY KEY NOT NULL,
+  "name" varchar(255) UNIQUE NOT NULL,
+  "user_id" int NOT NULL,
```
Comment (khudayberdiyev25) [inline migrations/000002_shop.up.sql:4]: I think better to rename `user_id` to `owner_id`. owner_id sounds more meaningful at shop context.

File: docker-compose.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ services:
       - $GOPATH/pkg/mod/cache:/go/pkg/mod/cache
     working_dir: /app
     ports:
-      - "5005:5005"
+      - "5007:5007"
```
Comment (khudayberdiyev25) [inline docker-compose.yaml:15]: why u changed it ?

**Review by khudayberdiyev25 — CHANGES_REQUESTED** [review]

File: migrations/000001_users.down.sql (hunk 1 above)
Comment (asadbek21iman) [reply migrations/000001_users.down.sql:1]: just empty line?

**Pull request closed without merging**